website/build_site.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/sh
 
 cp -r ../documentation content/
-jupyter nbconvert ../source/Mlos.Notebooks/*.ipynb --to markdown --output-dir content/notebooks
+jupyter nbconvert ../source/Mlos.Notebooks/*.ipynb --to markdown --output-dir content/notebooks --template nbconvert_template.md.j2
 
 # nbconvert and hugo disagree about paths
+# this should probably be done via the template
 sed -i 's/BayesianOptimization_files/\.\.\/BayesianOptimization_files/g' content/notebooks/BayesianOptimization.md
 
+# place links to github in notebook files
+for f in content/notebooks/*.md; do
+    base=$(basename "$f" '.md') # gives '25' from '25.conf'
```

Comment:
nit: that comment is technically wrong
In this case it would still give `25.conf` since the second arg is `.md`